rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -27,15 +27,19 @@
 import co.rsk.peg.whitelist.OneOffWhiteListEntry;
 import co.rsk.peg.whitelist.UnlimitedWhiteListEntry;
 import org.apache.commons.lang3.tuple.Pair;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
 import org.ethereum.core.Repository;
 import org.ethereum.vm.DataWord;
 
 import java.io.IOException;
 import java.util.*;
 
+import static org.ethereum.config.blockchain.upgrades.ConsensusRule.RSKIP123;
+import static org.ethereum.config.blockchain.upgrades.ConsensusRule.RSKIP87;
+
 /**
  * Provides an object oriented facade of the bridge contract memory.
- * @see co.rsk.remasc.RemascStorageProvider
+ * @see co.rsk.peg.BridgeStorageProvider
```

Comment:
References itself?

Reply:
good catch, I'll resolve this in a separated PR though.